### PR TITLE
seo optimisation

### PR DESCRIPTION
**Pull request by ntorbinskiy**
seo optimisation of AlexCode

### CODE REVIEW

**Review by 1alexvash — CHANGES_REQUESTED** [review]
Waiting for the update in this pull request

File: components/Post/PostContent.tsx [hunk 1]
```diff
@@ -16,22 +16,22 @@ const getFirstParagraph = (str: string) => {
 
 const PostContent = ({ post }: Props) => {
   const description = getFirstParagraph(post.content);
-
+//привет
```
[the remark this answers is not in the document]
Comment (1alexvash) [reply components/Post/PostContent.tsx:19]: ``
```suggestion
```

File: components/Post/PostContent.tsx [hunk 2]
```diff
@@ -16,22 +16,22 @@ const getFirstParagraph = (str: string) => {
...
         <title>{post.title}</title>
-        <meta name="description" content={config.site_description} />
-        <meta property="og:title" content={config.site_keywords[0]} />
+        <meta property="og:title" content={post.title} />
         <meta property="og:description" content={description} />
         <meta property="og:url" content={config.host_url} />
         <meta property="og:type" content="article" />
         <meta property="og:site_name" content={config.site_title} />
         <meta property="og:image" content={post.featuredImage} />
+        <meta name="description" content={config.site_description} />
```
[the remark this answers is not in the document]
Comment (ntorbinskiy) [reply components/Post/PostContent.tsx:30]: Okey, Which description would you like to see?

File: pages/index.tsx [hunk 3]
```diff
@@ -38,13 +38,13 @@ const Home: NextPage<{ posts: PostDocumentWithoutContent[] }> = ({ posts }) => {
     <>
       <Head>
         <title>{config.site_title}</title>
-        <meta name="description" content={config.site_description} />
         <meta property="og:title" content={config.site_keywords[1]} />
```
[the remark this answers is not in the document]
Comment (ntorbinskiy) [reply pages/index.tsx:41]: Are you talking about "og:title", am I right?

File: pages/index.tsx [hunk 4]
```diff
@@ -38,13 +38,13 @@ const Home: NextPage<{ posts: PostDocumentWithoutContent[] }> = ({ posts }) => {
...
         <meta property="og:description" content={config.site_description} />
         <meta property="og:url" content={config.host_url} />
         <meta property="og:type" content="website" />
         <meta property="og:site_name" content={config.site_title} />
         <meta property="og:image" content={config.defaultImage} />
+        <meta name="description" content={config.site_description} />
```
[the remark this answers is not in the document]
Comment (ntorbinskiy) [reply pages/index.tsx:47]: Prettier automatically deletes break line after meta tags. I turned off ,,Format on Save'' to save this break line

File: pages/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (1alexvash) [reply pages/index.tsx:41]: @ntorbinskiy 
Yeah, as it is a comment under line 42

File: pages/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (1alexvash) [reply pages/index.tsx:47]: @ntorbinskiy 
Animation
What do you mean by saying prettier deletes break lines?
It works for me perfectly

[the remark this answers is not in the document]
Comment (ntorbinskiy) [reply pages/index.tsx:47]: https://www.loom.com/share/98f6cd909221480499e0424aab17ea52

File: pages/index.tsx (hunk 3 above)
[the remark this answers is not in the document]
Comment (ntorbinskiy) [reply pages/index.tsx:41]: ### Main Ideas

1. Keep OG title short (without a brand name) with 60 characters for desktop and 40 for mobile.
2. Title Tags Should Contain Your Main Keyword
3. For best results, try to put your focus keyword as close to the beginning of your title as possible. That’s so search engines (and people) will see the keyword early on.
4. Make sure the keyword placement is organic. It’s preferable that the keyword is close to the beginning, but it’s not necessary for great SEO.
5. Much like a headline, a title tag needs to communicate a benefit to stand out.
6. Google suggests using your homepage title tag to include the most branding.
7.For most of your pages, adding your brand to the end of the title tag will suffice
8.Avoid Duplicate Title Tags

**Conclusion:**

### How do you write a good title tag?
Pay attention to the length, use the main keyword the page targets, and explain what benefit the user will get by clicking.

### How long should title tags be?
Between 50 and 50 characters. Any longer than that and Google may truncate your title.

### How many keywords should be in my title tags?
Generally just one. You can add a second if it is closely related and makes sense. Don’t keyword stuff; the goal of the title tag is to explain what users can expect

[the remark this answers is not in the document]
Comment (ntorbinskiy) [reply pages/index.tsx:41]: https://neilpatel.com/blog/open-graph-meta-tags/
https://neilpatel.com/blog/title-tags-seo/
https://magefan.com/blog/open-graph-meta-tags

File: pages/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (1alexvash) [reply pages/index.tsx:47]: @ntorbinskiy Um, funny

Do we have the same formatting settings, and user the same formatter?

https://i.stack.imgur.com/XohQa.png

File: components/Post/PostContent.tsx [hunk 5]
```diff
@@ -16,22 +16,28 @@ const getFirstParagraph = (str: string) => {
...
+		
       </Head>
 
       <div className="blogpost-image">
-        <img src={post.featuredImage} alt="blog post image" />
+        <img
+          src={post.featuredImage}
+          alt="blog post image"
+          width={790}
+          height={300}
```
Comment (1alexvash) [inline components/Post/PostContent.tsx:39]:
```suggestion
          height={394}
```

File: components/Post/PostContent.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (1alexvash) [reply components/Post/PostContent.tsx:30]: @ntorbinskiy Nothing, sorry, this one is fine, I thought you did edit the page description

File: pages/index.tsx (hunk 4 above)
[the remark this answers is not in the document]
Comment (1alexvash) [reply pages/index.tsx:47]: @ntorbinskiy What's the difference between line 42 and 47?

[the remark this answers is not in the document]
Comment (ntorbinskiy) [reply pages/index.tsx:47]: Yes, we do

**Pull request closed without merging**